### PR TITLE
Travis CI: Lint for Python syntax errors and undefined names

### PR DESCRIPTION
**Pull request by cclauss**
There are a few Python 3 incompatibilities to work on.

Thanks for taking the time to contribute to Git! Please be advised that the
Git community does not use github.com for their contributions. Instead, we use
a mailing list (git@vger.kernel.org) for code submissions, code reviews, and
bug reports. Nevertheless, you can use GitGitGadget (https://gitgitgadget.github.io/)
to conveniently send your Pull Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

**Comment by dscho** [comment]
@cclauss thank you!

The remaining [Travis issue](https://travis-ci.org/git/git/jobs/542825660) is a known one, and I think it will be addressed at some stage via the Git mailing list.

Back to your patches: you are probably a lot more familiar with Python than I am... from my side, the code changes look good.

Would you mind adding a bit of information to the commit messages (see e.g. https://github.com/git/git/commit/2000ac9fbf074e32cfd11109f6148a0c34d1133a for an example of a commit message in `.travis.yml`'s history), as well as your [sign off](https://git-scm.com/docs/SubmittingPatches#dco)?

Please make sure that your commit message focuses on answering the question "why?" more than on "how?", and that it wraps at <= 76 columns per line.

Finally send the patch to the mailing list for review. You can use [GitGitGadget](https://gitgitgadget.github.io), [submitGit](https://submitgit.herokuapp.com) or [send it manually](https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md#submit-your-patch).

File: contrib/fast-import/import-zips.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     exit(1)
 
 if len(argv) < 2:
-    print 'usage:', argv[0], '<zipfile>...'
+    print('usage:', argv[0], '<zipfile>...')
```
Comment (dilijev) [inline contrib/fast-import/import-zips.py]: Looks like this file needs `from __future__ import print_function` in case it is run from python 2

Comment (cclauss) [reply contrib/fast-import/import-zips.py]: Done.

**Comment by dscho** [comment]
@cclauss gentle ping?

**Comment by dscho** [comment]
Ah. This PR is superseded by https://github.com/gitgitgadget/git/pull/290.

**Pull request closed without merging**